fontforge/sfd.c

```diff
@@ -5848,8 +5848,6 @@ static int SFDGetBitmapChar(FILE *sfd,BDFFont *bdf) {
     EncMap *map;
     int ch;
 
-    bfc = chunkalloc(sizeof(BDFChar));
-    memset( bfc,'\0',sizeof( BDFChar ));
```

Comment:
This memset isn't needed because chunkalloc boils down to calloc which zeroes bfc